playwright-e2e/tests/singular/enrollment/adult-self-enrollment.spec.ts

```diff
@@ -76,12 +76,6 @@ test.describe('Enroll myself as adult', () => {
     await aboutMePage.waitForReady();
     await assertActivityHeader(page, 'About Me');
     await enterMailingAddress(page, { fullName: `${user.patient.firstName} ${lastName}` });
-    // Clicking of Next button Triggered address validation
-    await aboutMePage.next();
-    // Because address is all fake, an error message is expected
-    await expect(page.locator('.ErrorMessage')).toBeVisible();
-    await expect(page.locator('.ErrorMessage')).toHaveText(/We could not find the entered address/);
```

Comment:
Boilerplate code no longer needed because it's handled in `enterMailingAddress()`.